x-pack/plugins/rule_registry/kibana.json

```diff
@@ -2,15 +2,8 @@
   "id": "ruleRegistry",
   "version": "8.0.0",
   "kibanaVersion": "kibana",
-  "configPath": [
-    "xpack",
-    "ruleRegistry"
-  ],
-  "requiredPlugins": [
-    "alerting",
-    "data",
-    "triggersActionsUi"
-  ],
+  "configPath": ["xpack", "ruleRegistry"],
+  "requiredPlugins": ["alerting", "data", "triggersActionsUi"],
```

Comment:
Should this config also include `"owner"` field since the team has been assigned in the `CODEOWNERS`?

```
# Virtual teams
/x-pack/plugins/rule_registry/ @elastic/rac
```

Reply:
Thanks, nice catch! Done!

Reply:
I don't know what happened here, but I'll add the owner attribute in my next PR. The nice aspect about ci checks, no mistakes like this!